crates/bevy_color/src/laba.rs

```diff
@@ -110,6 +111,35 @@ impl Alpha for Laba {
     }
 }
 
+impl ClampColor for Laba {
+    fn clamped(&self) -> Self {
+        Self {
+            lightness: self.lightness.clamp(0., 1.5),
+            a: self.a.clamp(-1.5, 1.5),
+            b: self.b.clamp(-1.5, 1.5),
```

Comment:
This is consistent with the docs for the `a` and `b` fields, but I would argue that the docs are wrong here. In Lab, the `a` and `b` are unbounded and their practical ranges depend on the reference white. Similarly, the lightness is also only lower-bounded by 0, but technically any lightness can be expressed in Lab. Where 100 is defined as reference white, you *can* have brighter colors than your reference, so an upper bound does not make sense.

Some software clamp the a and b values to the `-128..127` range as they would work in signed 8-bit integers, but this is technically clipping the Lab color space (note that [`palette`](https://docs.rs/palette/latest/palette/struct.Lab.html) uses this anyway as their bounds for a and b).

I don't know what sources the documentation used to determine the original ranges of the fields, and it's possible that they actually referred to a variant of Lab; but it stands to reason that the definition for this color type should follow the most generally used variant of the color space (that is, in its full name, `L*a*b* CIE 1976`), in which case the only meaningful bound to check is `lightness > 0`.

Reply:
I don't know much about this colorspace in specific, and neither  Oklaba, but like you said, this is consistent with docs and what this trait should do: Ensure all the fields are within a given range. Changing this would require to change the range of those colorspaces declared in the docs, which is probably out of scope of this PR.

Reply:
I'm content to merge this as is and then fix it in a follow-up. I just don't want to end up in a place where we have inconsistent code and docs.

Reply:
Ok, let's go with that then!